QSB/AnglerFish/Events/AnglerResyncMessage.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using QSB.WorldSync.Events;
+using QuantumUNET.Transport;
+using UnityEngine;
+
+namespace QSB.AnglerFish.Events {
+    public class AnglerResyncMessage : WorldObjectMessage {
```

Comment:
please use multiple events/messages for this - means you don't have to write custom deserializers